pages/code_schools.js

```diff
@@ -285,7 +285,7 @@ function CodeSchools({ allSchools }) {
         <>
           <h3>{locationsModalInfo.name} Campuses</h3>
           {locationsModalInfo.locations.map(location => (
-            <div className={styles.schoolLocalModalItem}>
+            <div key={location.city} className={styles.schoolLocalModalItem}>
```

Comment:
this is something I also noticed. Let me know if you want to take care of separately or not.

Reply:
Yep. Good eyes. I fixed it in https://github.com/OperationCode/front-end/pull/1270